src/api/generator/findCreateApp.ts

```diff
@@ -62,6 +62,12 @@ const isEndpointBuilderResult = (node: ts.Node, typeChecker: ts.TypeChecker) =>
   return true
```

Comment:
Should we stick to yarn or npm? I'm okay with either, but let's clean up package-lock.json or yarn.lock depending on what we choose

Reply:
I think we should stick with `npm`. I've never used `yarn` before so I'm not sure how compatiable it is with `npm`, but I think using `npm` might make it easier in the future if we plan to upload this as an `npm` module on the `npm` repository.